easyflash/src/ef_env.c

```diff
@@ -495,7 +495,11 @@ static uint32_t find_next_env_addr(uint32_t start, uint32_t end)
     for (; start < end; start += (sizeof(buf) - sizeof(uint32_t))) {
         ef_port_read(start, (uint32_t *) buf, sizeof(buf));
         for (i = 0; i < sizeof(buf) - sizeof(uint32_t) && start + i < end; i++) {
+#if EF_LITTLE_ENDIAN            // Little Endian Order
```

Comment:
多谢贡献哈。要不改为 

```suggestion
#ifndef EF_BIG_ENDIAN            /* Little Endian Order */
```

毕竟小端更加常见，这样 ef_cfg.h 很多都不用改了

Reply:
嗯 挺好~